src/components/StyledForm.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore';
+
+export default function StyledForm() {
+	const [inputValue, setInputValue] = useState('');
+
+	const items = useStore(state => state.items);
+	const addItem = useStore(state => state.addItem);
+
+	return (
+		<>
+			<ul>
+				{items.map(item => {
+					return <li key={item.id}>{item.item}</li>;
+				})}
+			</ul>
+			<form
+				aria-label="Add a item"
+				onSubmit={event => {
+					event.preventDefault();
+					addItem(inputValue);
+					setInputValue('');
+				}}
+			>
+				<label htmlFor="item">
+					Item:
+					<StyledInput
+						id="item"
+						type="text"
+						required
+						minLength="3"
+						placeholder="don't forget me"
+						value={inputValue}
+						onChange={event => {
+							setInputValue(event.target.value);
+						}}
```

Comment:
In my opinion you do not need to save the user input with the `onChange` event to a state. If I understand correctly, you want the user to fill out the form and once the form is submitted the item will be saved to the list. 

Please have a look at this [CodeSandbox](https://codesandbox.io/s/usestate-form-exercise-solution-dg8d9f) from your **JS Form Session**. I suggest to update your form according to that example. In this way you can get rid of the `inputValue` state.

Reply:
Thanks for your feedback :) what is the advantage of not using the useState? I tried it once as in your linked codesandbox example with the handleSubmit function. Here it is unfortunately not so easy to reset the input field after the submit. Also in the linked example this is not implemented. Since I would like to have this function, I add this point to the acceptance criteria in the user story

Reply:
Actually it is easy :) have a look here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset

You can try it out in the CodeSandbox first before you implement it in your project. 🙂